.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "cvs-nop"]
```

Comment:
Ah so we went down the submodule route ... 
I think it will be good to have a DVSA stance on this @SanjeetBJSS  
and bring some best practice advice in for other team members

Reply:
Given that this services only job is to take a Dynamo Stream and convert it to the RDS's table structure ensuring that the structure is accurate and up to date is imperative . If you would like, I can get Paul to copy and paste the files in. I used to use the copy and paste approach in c#, when plugging WCF DTO's into WPF but given the nature of this service, and the simplicity of the SQL script, it kinda made sense. There is no submodules or dependancies below it, and no chance of getting into a web of dependancy. If you would like Paul to copy it in, I can ask him to.

Reply:
Have others in development worked on this and are comfortable it is workable - if @JonLazariniDD  etc say that is fine. I am OK if it is working

Reply:
Paul & I had a catch up yesterday about this and clarified in the README an additional step (where the branch of the sql repo with the changes had to be checked out and I had this step wrong).
I think it has been implemented (and automated) in the pipeline so it can be a good approach for now and use an alternative approach later should we think it is required.